java/src/Ice/src/main/java/com/zeroc/Ice/UserException.java

```diff
@@ -6,12 +6,6 @@
 
 /** Base class for exceptions defined in Slice. */
 public abstract class UserException extends java.lang.Exception implements Cloneable {
-  public UserException() {}
-
-  public UserException(Throwable cause) {
-    super(cause);
-  }
```

Comment:
The `Throwable` constructor was removed.
And in Java, if you don't define constructors for a class, you will always implicitly get:
```
public UserException() {  super(); }
```
So, there's no reason to write the 'default' constructor out here.